devices.js

```diff
@@ -17319,27 +17319,28 @@ const devices = [
     },
 
     {
-        zigbeeModel: ['HAN'],
-        model: 'Datek_HAN',
+        zigbeeModel: ['Meter Reader'],
+        model: 'Meter_Reader',
```

Comment:
Having `Meter_Reader` is not an acceptable model number, it should be something that identifies the devices (preferably something that is googleable)

Reply:
@Koenkk How can I change this? I can urge the developers at the manufacturer to change this in a future fw update. `Meter_Reader` is what are announced by the device... Do you have any suggestions?

Reply:
@TheStigh model can be freely chosen, only zigbeeModel has to match what the device announces.